src/App.jsx

```diff
@@ -1,11 +1,49 @@
 import './App.css';
-import { ChakraProvider, Text } from '@chakra-ui/react'
+// import from 'react';
+// import { ChakraProvider, Text } from '@chakra-ui/react'
+import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
+import { CookiesProvider } from 'react-cookie';
+import Login from './components/Login';
+import Logout from './components/Logout';
+import Register from './components/Register/register';
+import Dashboard from './pages/Dashboard/Dashboard';
+import ForgotPassword from './components/ForgotPassword';
+import EmailAction from './components/EmailAction';
+import AUTH_ROLES from './utils/auth_config';
+import ProtectedRoute from './utils/ProtectedRoute';
+
+const { ADMIN_ROLE, USER_ROLE } = AUTH_ROLES.AUTH_ROLES;
 
 const App = () => {
   return (
-    <ChakraProvider>
+    <div>
+      {/* <ChakraProvider>
       <Text>Hello World</Text>
-    </ChakraProvider>
+    </ChakraProvider> */}
```

Comment:
could you put all the new code inside `ChakraProvider`? that way in the future we can style with chakraUI